data/json/recipes/recipe_electronics.json

```diff
@@ -2215,6 +2215,39 @@
   ]
 },{
   "type" : "recipe",
+  "result": "hdcamera",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_PARTS",
+  "skill_used": "mechanics",
+  "skills_required": [ "electronics", 4 ],
+  "difficulty": 7,
+  "time": 90000,
+  "book_learn": [ ["textbook_robots", 6] ],
+  "autolearn": false,
+  "qualities":[
+    {"id":"SCREW_FINE","level":1},
+    {"id":"SAW_M","level":1}
+  ],
+  "tools": [
+    [
+      [ "toolset", 10 ]
```

Comment:
This is the actually the bionic toolset so you might want to add the soldering iron also.